test/components.ts

```diff
@@ -16,16 +18,36 @@ import { initComponents as originalInitComponents } from "../src/components"
  */
 export const test = createRunner<TestComponents>({
   main,
-  initComponents,
+  initComponents
 })
 
 async function initComponents(): Promise<TestComponents> {
   const components = await originalInitComponents()
 
-  const { config } = components
+  const { logs, database } = components
+
+  const config = await createDotEnvConfigComponent({ path: ['.env.default', '.env'] })
+
+  const fetch = {
+    async fetch(url: nodeFetch.RequestInfo, init?: nodeFetch.RequestInit): Promise<nodeFetch.Response> {
+      return nodeFetch.default(url, init).then(async (response: nodeFetch.Response) => {
+        if (response.ok) {
+          // response.status >= 200 && response.status < 300
+          return response
+        }
+
+        throw new Error(await response.text())
+      })
+    }
+  }
+
+  const sceneAdminManager = await createSceneAdminManagerComponent({ database, logs })
```

Comment:
this ain't being already created in normal components.ts? do we need to recreate it?